less/variables.less

```diff
@@ -1,8 +1,8 @@
 @big: ~"only screen and (min-width: 1400px) and (max-width: 3000px)";
 @desktop: ~"only screen and (min-width: 1100px) and (max-width: 1399px)";
-@tablet: ~"only screen and (min-width: 620px) and (max-width: 1099px)";
-@phone: ~"only screen and (max-width: 619px)";
-@phone-max: 620px;
+@tablet: ~"only screen and (min-width: 701px) and (max-width: 1099px)";
+@phone: ~"only screen and (max-width: 700px)";
+@phone-max: 700px;
```

Comment:
How and why did you decide thoses limits? I understand this is based on #1248 but to me it seems like a rule of thumb rather than based on a global view.